cpp/open3d/geometry/TriangleMesh.cpp

```diff
@@ -386,6 +396,16 @@ std::shared_ptr<TriangleMesh> TriangleMesh::FilterSmoothLaplacian(
             std::swap(mesh->vertex_colors_, prev_vertex_colors);
         }
     }
+    if (filter_vertex) {
```

Comment:
Is this a bugfix? Can you move it to a separate PR, since its unrelated to the filament upgrade?